src/Microsoft.Extensions.Caching.Memory/project.json

```diff
@@ -23,7 +23,11 @@
     "Microsoft.Extensions.Options": "1.0.0-*"
   },
   "frameworks": {
-    "net451": {},
+    "net451": {
```

Comment:
Thinking more I suppose you are being forced to use 1.3 as the dependencies require it. You can ignore my comment.